libraries/AP_Math/AP_Math.cpp

```diff
@@ -13,7 +13,7 @@ template <class Arithmetic1, class Arithmetic2>
 typename std::enable_if<std::is_floating_point<typename std::common_type<Arithmetic1, Arithmetic2>::type>::value, bool>::type
 is_equal(const Arithmetic1 v_1, const Arithmetic2 v_2)
 {
-    return fabsf(v_1 - v_2) < std::numeric_limits<decltype(v_1 - v_2)>::epsilon();
+    return fabsf(v_1 - v_2) < std::numeric_limits<float>::epsilon();
```

Comment:
Maybe we shouldn't use *fabsf* because this can be used for doubles (the function is, at least, instantiated for them).